dashboard/config/application.rb

```diff
@@ -100,8 +100,6 @@ class Application < Rails::Application
       jquery.handsontable.full.css
       jquery.handsontable.full.js
     )
-    config.react.variant = :development
```

Comment:
I _think_ chef handles NODE_ENV for us (we reference it in a couple of our Chef scripts).  @wjordan can you confirm that?  That said, I suppose we'll want a more explicit 'build production' argument since we expect to never build this package in the actual production environment.